pages/common/dash.md

```diff
@@ -7,25 +7,25 @@
 
 `dash`
 
-- Execute a [c]ommand:
+- Execute [c]ommands:
 
-`dash -c "{{command}}"`
+`dash -c "echo 'dash is executed'"`
```

Comment:
Missing token syntax. This goes for all variants of this example. Also, I've adjusted the example echo message - but this is just a cosmetic change suggestion.
```suggestion
`dash -c "{{echo 'hello, world from dash'}}"`
```

Reply:
I don't understand why braces are required around the sample command. There is no placeholder identifier such `{{command}}` here.

Reply:
Done for all pages.